images/kubekins-e2e/variants.yaml

```diff
@@ -17,7 +17,7 @@ variants:
     BAZEL_VERSION: 0.23.2
   '1.16':
     CONFIG: '1.16'
-    GO_VERSION: 1.12.12
+    GO_VERSION: 1.13.4
```

Comment:
oh, I think we need a rebuild/bump of kubekins images to pick this up

Reply:
@cblecker @BenTheElder is there an automated process to bump these or do I need to open a PR?

Reply:
looks like https://github.com/kubernetes/test-infra/pull/15627 will do that

Reply:
The bump process is still somewhat manual, but will be merged later today.